src/content/basics/automatic-termination-of-bad-nodes/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Automatic termination of bad nodes
+description: A general description of a feature that can detect and terminate unhealthy nodes in the tenant cluster.
+date: 2020-11-05
+weight: 120
+type: page
+categories: ["basics"]
+---
+
+# Automatic termination of unhealthy nodes
+
+## Overview
+
+Giant Swarm's `terminate-unhealthy-nodes` feature helps you keep the nodes in your cluster in a healthy, running state. When enabled, a service makes periodic checks on the health state of each node in your cluster. If a node fails consecutive health checks over an extended time period, it will be drained and terminated.
```

Comment:
```suggestion
Giant Swarm's `terminate-unhealthy-nodes` feature helps you keep the nodes in your cluster in a healthy, running state. When enabled, a service makes periodic checks on the health of each node in your cluster. If a node fails consecutive health checks over an extended time period, it will be drained and terminated.
```